flexmeasures/api/v3_0/tests/test_sensor_schedules.py

```diff
@@ -28,14 +28,15 @@
             "Not a valid number",
         ),
         (message_for_trigger_schedule(), "soc-unit", "MWH", "Must be one of"),
-        (
-            message_for_trigger_schedule(
-                with_targets=True, too_far_into_the_future_targets=True
-            ),
-            "soc-targets",
-            None,
-            "Target datetime exceeds",
-        ),
+        # todo: add back test in case we stop grandfathering ignoring too-far-into-the-future targets, or amend otherwise
```

Comment:
I guess it would be correct to say "... in case we stop grandfathering too-far-into-the-future targets". The "ignoring" *is* the grandfathering.